lib/src/util/flutter_utils.dart

```diff
@@ -13,9 +14,8 @@ var _collectionInterfaces = <InterfaceTypeDefinition>[
   InterfaceTypeDefinition('LinkedHashSet', 'dart.collection'),
 ];
 
-// todo (pq): consider caching lookups
-bool isWidgetType(DartType type) =>
-    DartTypeUtilities.implementsInterface(type, 'Widget', '');
+bool isContainerType(DartType type) =>
+    DartTypeUtilities.implementsInterface(type, 'Container', '');
```

Comment:
Is this the semantic you want - not just exactly `Container`, but also its subclasses?
Why not `isInterface`?

Reply:
See for example `isExactWidgetTypeContainer` is DAS.

Reply:
hmm, yes.  i think you're right.  thanks.

Reply:
Ok, fixed.  PTAL?